settings.gradle

```diff
@@ -1,4 +1,11 @@
+rootProject.name = 'okhttp-parent'
+
 include ':mockwebserver'
+if (!properties.containsKey('android.injected.invoked.from.ide') && System.getenv(
```

Comment:
@swankjesse @JakeWharton  This should make the code pretty safe for both normal development and also CI.  But allow us to do real local tests against phones etc to flush out bugs.

Thoughts?

Reply:
Works for me. Got an issue to link to so we know when we can delete this?